src/main/resources/swagger.api/dcb_transaction.yaml

```diff
@@ -6,148 +6,124 @@ info:
-        schema:
-          type: string
     post:
       description: Create circulation request
       operationId: createCirculationRequest
       tags:
         - circulation
+      parameters:
+        - $ref: '#/components/parameters/dcbTransactionId'
       requestBody:
-        content:
-          application/json:
-            schema:
-              type: object
-              properties:
-                item:
-                  $ref: '#/components/schemas/DCBItem'
-                patron:
-                  $ref: '#/components/schemas/DCBPatron'
+        $ref: "#/components/requestBodies/DCBTransaction"
       responses:
         '201':
-          description: Accepted
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/TransactionStatusResponse'
-
+          $ref: '#/components/responses/TransactionStatusResponse'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '409':
+          $ref: '#/components/responses/Conflict'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
   /transactions/{dcbTransactionId}/status:
     parameters:
-      - in: path
-        name: dcbTransactionId
-        description: The ReShare DCB ID for the transaction being brokered.
-        required: true
-        schema:
-          type: string
+      - $ref: '#/components/parameters/dcbTransactionId'
     get:
       description: Get transaction status across circulation institutions
       operationId: getTransactionStatus
       tags:
         - circulation
       responses:
         '200':
-          description: Status updated
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/TransactionStatusResponse'
+          $ref: '#/components/responses/TransactionStatusResponse'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
     put:
       description: Update transaction status across circulation institutions
       operationId: updateTransactionStatus
       tags:
         - circulation
       requestBody:
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/TransactionStatusResponse'
+        $ref: '#/components/requestBodies/TransactionStatusBody'
       responses:
         '200':
-          description: Status updated
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/TransactionStatusResponse'
-
+          $ref: '#/components/responses/TransactionStatusResponse'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
```

Comment:
I am just thinking upon if there are any more error codes that we should add. @kalibek-epam  has K-int provided a list of error codes that we have to match up with or these many are enough at the moment ?